scarb/src/core/manifest/mod.rs

```diff
@@ -148,6 +148,7 @@ pub struct ManifestMetadata {
     pub repository: Option<String>,
     #[serde(rename = "tool")]
     pub tool_metadata: Option<BTreeMap<SmolStr, Value>>,
+    pub cairo_version: Option<Version>,
```

Comment:
This should be `VersionReq` isn't it? We want to allow users specifying ranges of Cairo versions

```suggestion
    pub cairo_version: Option<VersionReq>,
```